maven_iuvs/echelle.py

```diff
@@ -657,21 +657,43 @@ def subtract_darks(light_fits, dark_fits):
     first_dark = darks[0, :, :]
     second_dark = darks[1, :, :]
 
-    # check darks are valid
-    if np.isnan(first_dark).any() & np.isnan(second_dark).any():
-        raise Exception(f"Missing critical observation data: no valid darks")
-
```

Comment:
Please note this logic is still present, see code at right, line 694

Reply:
line numbers don't match but I see code below under the comment `CHECK VALIDITY OF DARKS`